tests/functional/spec/introduction.spec.js

```diff
@@ -12,8 +12,5 @@ describe("Introduction page", () => {
     expect($(IntroductionPage.introDescription()).getText()).to.contain(
       "To take part, all you need to do is check that you have the information you need to answer the survey questions."
     );
-    expect($(IntroductionPage.introTitleDescription()).getText()).to.contain(
```

Comment:
Oddly name, but as we have absorbed this now into content the data-qa doesn't exist. You could argue we should still test for this, but after a quick look around we don't seem to test like this anywhere else for placeholders, just integration